packages/wrangler/src/__tests__/cloudchamber/curl.test.ts

```diff
@@ -0,0 +1,126 @@
+import { useMockIsTTY } from "../helpers/mock-istty";
+import { msw } from "../helpers/msw";
+import { runInTempDir } from "../helpers/run-in-tmp";
+import { runWrangler } from "../helpers/run-wrangler";
+import { mockAccount, setWranglerConfig } from "./utils";
+
+describe("cloudchamber curl", () => {
+	const std = mockConsoleMethods();
+	const { setIsTTY } = useMockIsTTY();
+
+	mockAccountId();
+	mockApiToken();
+	runInTempDir();
+	beforeEach(mockAccount);
+
+	afterEach(() => {
+		patchConsole(() => {});
+		msw.resetHandlers();
+	});
+
+	it("should help", async () => {
+		await runWrangler("cloudchamber curl --help");
+		expect(std.err).toMatchInlineSnapshot(`""`);
+		expect(std.out).toMatchInlineSnapshot(`
+			"wrangler cloudchamber curl <path>
+
+			perform curl in wrangler
+
+			POSITIONALS
+			  path  [string] [required] [default: \\"/\\"]
+
+			GLOBAL FLAGS
+			  -j, --experimental-json-config  Experimental: support wrangler.json  [boolean]
+			  -c, --config                    Path to .toml configuration file  [string]
+			  -e, --env                       Environment to use for operations and .env files  [string]
+			  -h, --help                      Show help  [boolean]
+			  -v, --version                   Show version number  [boolean]
+
+			OPTIONS
+			      --json                Return output as clean JSON  [boolean] [default: false]
+			  -H, --header              Add headers in the form of --header <name>:<value>  [array]
+			  -D, --data                Add a JSON body to the request  [string]
+			  -X, --method  [string] [default: \\"GET\\"]
+			  -s, --silent              Only output response  [boolean]
+			  -v, --verbose             Show version number  [boolean]
+			      --use-stdin, --stdin  Equivalent of using --data-binary @- in curl  [boolean]"
+		`);
+	});
+
+	it("should be able to use data flag", async () => {
+		setIsTTY(false);
+		setWranglerConfig({});
+		msw.use(
+			http.post("*/deployments/v2", async ({ request }) => {
+				expect(await request.text()).toMatchInlineSnapshot(
+					`"{\\"image\\":\\"hello:world\\",\\"location\\":\\"sfo06\\",\\"ssh_public_key_ids\\":[],\\"environment_variables\\":[{\\"name\\":\\"HELLO\\",\\"value\\":\\"WORLD\\"},{\\"name\\":\\"YOU\\",\\"value\\":\\"CONQUERED\\"}],\\"vcpu\\":3,\\"memory\\":\\"400GB\\",\\"network\\":{\\"assign_ipv4\\":\\"predefined\\"}}"`
+				);
+				return HttpResponse.json(MOCK_DEPLOYMENTS_COMPLEX[0]);
+			})
+		);
+
+		// We need to stringify this for cross-platform compatibility
+		const deployment = {
+			image: "hello:world",
+			location: "sfo06",
+			ssh_public_key_ids: [],
+			environment_variables: [
+				{ name: "HELLO", value: "WORLD" },
+				{ name: "YOU", value: "CONQUERED" },
+			],
+			vcpu: 3,
+			memory: "400GB",
+			network: { assign_ipv4: "predefined" },
+		};
+
+		await runWrangler(
+			`cloudchamber curl /deployments/v2 --json -X POST -D '` +
+				JSON.stringify(deployment) +
+				`'`
+		);
```

Comment:
NIT given you are using backticks then you may as well interpolate.

```suggestion
		await runWrangler(
			`cloudchamber curl /deployments/v2 --json -X POST -D '${JSON.stringify(deployment)}'`
		);
```